templates/zerver/integrations/capistrano.md

```diff
@@ -14,26 +18,24 @@ Get Zulip notifications for your Capistrano deploys!
           desc "Post a message to Zulip after deploy"
           task :humbug do
             run_locally "echo 'I just deployed to #{stage}! :tada:' | zulip-send \
-            --user capistrano-bot@example.com --api-key a0b1c2d3e4f5a6b7c8d9e0f1a2b3c4d5 \
+            --user <email of bot> --api-key a0b1c2d3e4f5a6b7c8d9e0f1a2b3c4d5 \
             --site={{ api_url }} \
             --stream commits --subject deployments || true"
           end
         end
 
-    The `--user` and `--api-key` should be the email and API key of the Zulip
-    bot created above. You can also put these values in a `~/.zuliprc` file on
-    your Capistrano machine. See our [API docs](/api/) for instructions on
-    creating that file.
+{end_tabs}
 
-!!! tip ""
+{!congrats.md!}
 
-    You can change the `deploy` above to another step of
-    your deployment process, if you'd like the notification to fire
-    at a different time. See [Capistrano's Before/After Hooks page][1]
-    for more information!
+![Capistrano bot message](/static/images/integrations/capistrano/001.png)
```

Comment:
I've opened an issue #32462 to track these updates. Would you still like me to post in CZO asking regarding their removal?

Reply:
Yes, let's ask in the documentation channel on CZO. I think since a) we're updating this doc and b) the screenshot is *very* out of date with the current UI, it might make sense to just remove it and file the issue for  updating the screenshot at a later date. But I think it makes sense to get other folks thoughts on that.